docs/latest/design/extending-envoy-gateway.md

```diff
@@ -50,24 +50,26 @@ An example configuration:
 ```yaml
 apiVersion: config.gateway.envoyproxy.io/v1alpha1
 kind: EnvoyGateway
-extension:
+extensionManager:
   resources:
   - group: example.myextension.io
     version: v2
     kind: OAuth2Filter
-  hooks:
-    post:
-    - Route
-    - VirtualHost
-    - HTTPListener
-    - Translation
-  service:
-    host: my-extension.example
-    port: 443
-    tls:
-      certificateRef:
-        name: my-secret
-        namespace: default
+  extensionManager:
```

Comment:
looks like an extra `extensionManager`
also indentation seems off for `hooks` and everything below it

Reply:
Fixed